static/app/views/issueDetails/streamline/flagSeries.tsx

```diff
@@ -65,6 +66,10 @@ export default function useFlagSeries({query = {}, event, group}: FlagSeriesProp
   }
 
   const hydratedFlagData = hydrateToFlagSeries(rawFlagData);
+  const intersectionFlagNames = getFlagIntersection({hydratedFlagData, event});
+  const intersectionFlags = hydratedFlagData.filter(f =>
+    intersectionFlagNames.includes(f.name)
+  );
```

Comment:
```suggestion
  const evaluatedFlagNames = event?.contexts.flags?.values.map(f => f.flag);
  const intersectionFlags = hydratedFlagData.filter(f =>
   !evaluatedFlagNames.includes(f.name)
  );
```
I think you can write it this way so the helper isn't needed

Reply:
Would any of these benefit from useMemo?

Reply:
should be `evaluatedFlagNames.includes(f.name)` right?

Reply:
> should be evaluatedFlagNames.includes(f.name) right?
Oops, yes!